VnE/Source/Extensions/ChemPack/parsers.py

```
@@ -502,50 +488,50 @@ def createCellList(self, parent, cell_dec_coords):
+        ca = point_class.PointsList(parent=cell_rend, name='c-a', color=cell_rend, rad=cell_rend)
+        point1, point2 = (point_class.Point(parent=ca, coord=c, color=ca, rad=ca),
+                          point_class.Point(parent=ca, coord=cell_dec_coords[6], color=ca, rad=ca))
+
+        abc = point_class.PointsList(parent=cell_rend, name='ab-c', color=cell_rend, rad=cell_rend)
+        point1, point2 = (point_class.Point(parent=abc, coord=cell_dec_coords[7], color=abc, rad=abc),
+                          point_class.Point(parent=abc, coord=cell_dec_coords[5], color=abc, rad=abc))
+        acb = point_class.PointsList(parent=cell_rend, name='ac-b', color=cell_rend, rad=cell_rend)
+        point1, point2 = (point_class.Point(parent=acb, coord=cell_dec_coords[7], color=acb, rad=acb),
+                          point_class.Point(parent=acb, coord=cell_dec_coords[6], color=acb, rad=acb))
+        bca = point_class.PointsList(parent=cell_rend, name='bc-a', color=cell_rend, rad=cell_rend)
+        point1, point2 = (point_class.Point(parent=bca, coord=cell_dec_coords[7], color=bca, rad=bca),
```

Ruff (0.14.14)

[error] Local variable `point1` is assigned to but never used

Remove assignment to unused variable `point1`

(F841)